xdsl/dialects/builtin.py

```diff
@@ -34,16 +34,6 @@ def parse_parameter(parser: BaseParser) -> str:
     def print_parameter(data: str, printer: Printer) -> None:
         printer.print_string(f'"{data}"')
 
-    @staticmethod
```

Comment:
The current StringAttr builder comes from dataclass, right?

Reply:
Yes!